swarmlib/__main__.py

```diff
@@ -72,6 +76,10 @@ def run_swarm():
     args = vars(parser.parse_args())
     if args:
         algorithm = args.pop('func')
+        loglevel = args.pop('log_level', "INFO")
+        level = getattr(logging, loglevel.upper(), None)
```

Comment:
The default value here should also be unnecessary when we use a _choice_ in the parser